signals/helpers.py

```diff
@@ -0,0 +1,16 @@
+import os.path
+import glob
+
+
+def recursively_find_parent_containing_file(current_path, search_files):
```

Comment:
* it goes up to the root IF it doesn't find what it's looking for on the way.  If it finds what it's looking for, it stops immediately.